irods_store.py

```diff
@@ -397,7 +412,7 @@ def delete(self, location):
 
```

Comment:
^^ LOG.debug(_("connecting to %(host)s for %(data)s" %
                   ({'host': self.host, 'data': full_data_path})))